html2asketch/nodeToSketchLayers.js

```diff
@@ -332,6 +332,10 @@ export default function nodeToSketchLayers(node, options) {
         multiline: numberOfLines > 1
       });
 
+      if (options && options.onTextGenerate) {
+        options.onTextGenerate({text, node: textNode.parentNode});
```

Comment:
nit how about we change `text,` with a `layer: text`? `text` makes sense in the context of this method, but outside someone can expect `text` to be a string which might be confusing.

Reply:
I've passed it `{layer: text, node: textNode}`, there might be cases where the full `textNode` object is needed.